python-flask/config.py

```diff
@@ -0,0 +1,6 @@
+import os
+
+SECRET_KEY = os.getenv('SECRET_KEY')
+SQLALCHEMY_DATABASE_URI = os.getenv('SQLALCHEMY_DATABASE_URI')
+SQLALCHEMY_TRACK_MODIFICATIONS = False
+PER_PAGE = 50
```

Comment:
I this PER_PAGE making reference to the amount of messages? if that's the case I think the number we defined in the specification is lower right? If we dont have specification for it we need to define. But for some reason I have in mind that is 30 or 20

Reply:
Yea. I can't recall this tbo. But let's define 30? I changed it to 30 and we can commit on that.